spec/computer_spec.rb

```diff
@@ -14,10 +14,6 @@
     expect(computer.get_move.nil?).to eq(false)
```

Comment:
how about `expect(computer.get_move).to be_nil` ?

https://relishapp.com/rspec/rspec-expectations/v/3-8/docs/built-in-matchers/be-matchers

Reply:
Thanks for this The test expectation has changed but I'm now using .not_to be_nil out of the be nil rspec matchers. #40